### PR TITLE
Refactoring methods and attributes to match smallCamelCase

### PR DESCRIPTION
**Pull request by JanVetter**
- [ ] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Ensure to use auto format in **all** files
- [ ] Tests created for changes or:
  - [ ] Test cases are missing - opened issue for that:
  - [ ] Reason for missing tests:
- [ ] Screenshots added (for UI changes)
- [ ] Change in CHANGELOG.md described

---

## Short Description

Refactoring methods and attributs to match smallCamelCase.

## Resolving Issue/ Feature

Fixes issue #92 .

### CODE REVIEW

File: mico-core/src/test/java/io/github/ust/mico/core/MicoCoreApplicationTests.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public void testDependencyServiceRepository(){
 		//2. service
         Service service2 = new Service(testShortName2,testVersion2);
         service2.setName(testLongerName);
-        service2.setDescription(testServiceDescription);
+        service2.setDescription(testserviceInterface);
```
Comment (Pascal157600) [inline mico-core/src/test/java/io/github/ust/mico/core/MicoCoreApplicationTests.java]: It should be "testServiceInterface", right?

File: mico-core/src/test/java/io/github/ust/mico/core/MicoCoreApplicationTests.java [hunk 2]
```diff
@@ -128,7 +128,7 @@ public void testDependencyServiceRepository(){
         assertNotNull(testService2);
         assertEquals(testVersion2,testService2.getVersion());
         assertEquals(testLongerName,testService2.getName());
-        assertEquals(testServiceDescription,testService2.getDescription());
+        assertEquals(testserviceInterface,testService2.getDescription());
```
Comment (Pascal157600) [inline mico-core/src/test/java/io/github/ust/mico/core/MicoCoreApplicationTests.java]: same here

File: mico-core/src/test/java/io/github/ust/mico/core/SpringDataRestTests.java [hunk 3]
```diff
@@ -48,28 +48,28 @@ public void testServicesWithMultipleDescriptions() throws Exception {
...
-        String serviceDescriptionPortVariable = "<PORT>";
-        String serviceDescriptionProtocol = "HTTP";
+        String serviceInterfaceValue = "Some description";
+        String serviceInterfacePortVariable = "<PORT>";
+        String serviceInterfaceProtocol = "HTTP";
         String serviceName = "Some Service Name";
 
-        ServiceInterface serviceInterface1 = generateServiceDescription("1");
-        ServiceInterface serviceInterface2 = generateServiceDescription("2");
+        ServiceInterface serviceInterface1 = generateserviceInterface("1");
```
Comment (Pascal157600) [inline mico-core/src/test/java/io/github/ust/mico/core/SpringDataRestTests.java]: and here

File: mico-core/src/test/java/io/github/ust/mico/core/SpringDataRestTests.java [hunk 4]
```diff
@@ -48,28 +48,28 @@ public void testServicesWithMultipleDescriptions() throws Exception {
...
+        ServiceInterface serviceInterface2 = generateserviceInterface("2");
 
-        URI serviceDescriptionLinkSelf1 = postServiceDescription(serviceInterface1);
-        URI serviceDescriptionLinkSelf2 = postServiceDescription(serviceInterface2);
+        URI serviceInterfaceLinkSelf1 = postserviceInterface(serviceInterface1);
+        URI serviceInterfaceLinkSelf2 = postserviceInterface(serviceInterface2);
 
-        putServiceDescription(serviceDescriptionsURI, serviceDescriptionLinkSelf1);
-        putServiceDescription(serviceDescriptionsURI, serviceDescriptionLinkSelf2);
+        putserviceInterface(serviceInterfacesURI, serviceInterfaceLinkSelf1);
```
Comment (Pascal157600) [inline mico-core/src/test/java/io/github/ust/mico/core/SpringDataRestTests.java]: here

File: mico-core/src/test/java/io/github/ust/mico/core/SpringDataRestTests.java [hunk 5]
```diff
@@ -78,20 +78,20 @@ public void testDeleteService() throws Exception {
         Service service = generateService("2");
         JsonNode serviceResult = postService(service);
         JsonNode links = serviceResult.get("_links");
-        JsonNode linkServiceDescriptions = links.get("serviceDescriptions").get("href");
+        JsonNode linkserviceInterfaces = links.get("serviceInterfaces").get("href");
```
Comment (Pascal157600) [inline mico-core/src/test/java/io/github/ust/mico/core/SpringDataRestTests.java]: here

**Comment by JanVetter** [comment]
@Pascal157600 Passt's jetzt?

**Pull request merged**